mix.exs

```diff
@@ -29,6 +29,7 @@ defmodule Tokenizers.MixProject do
 
   defp deps do
     [
+      {:dialyxir, "~> 1.3", only: [:test, :dev], runtime: false},
```

Comment:
I believe none of the maintainers use Dialyzer, can we remove it for now?

Reply:
It can be added to CI, but I'll just remove it then